features/steps/test_starter/{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,7 +1,41 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{ cookiecutter.python_package }}"
+readme = "README.md"
+dynamic = ["dependencies", "version"]
+
+[project.scripts]
+{{ cookiecutter.repo_name }} = "{{ cookiecutter.python_package }}.__main__:main"
+
+[project.optional-dependencies]
+docs = [
+    "docutils<0.18.0",
+    "sphinx~=3.4.3",
+    "sphinx_rtd_theme==0.5.1",
+    "nbsphinx==0.8.1",
+    "nbstripout~=0.4",
+    "sphinx-autodoc-typehints==1.11.1",
+    "sphinx_copybutton==0.3.1",
+    "ipykernel>=5.3, <7.0",
+    "Jinja2<3.1.0",
+    "myst-parser~=0.17.2",
+]
+
+[tool.setuptools.dynamic]
+dependencies = {file = "src/requirements.txt"}
```

Comment:
Decided not to move this file just yet, maybe too bold of a change... But maybe it's a good moment to change the `pip install -r src/requirements.txt` habit to just `pip install -r requirements.txt`? Please advise.

Reply:
And move `requirements.txt` up a level you mean? I don't quite remember why it was inside `src` tbh.. maybe it had to do with `kedro build-reqs`?

Reply:
Exactly, this would mean moving `requirements.txt` up one level. If we're okay with it, more than happy to proceed (having them inside `src` at this point would be extra weird)

Reply:
Done. There's quite a bit of churn, but it is what it is...